core-bundle/src/Altcha/Config/AlgorithmConfig.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Altcha\Config;
+
+enum AlgorithmConfig: string
+{
+    case ALGORITHM_SHA_256 = 'SHA-256';
+    case ALGORITHM_SHA_384 = 'SHA-384';
+    case ALGORITHM_SHA_512 = 'SHA-512';
```

Comment:
```suggestion
    case algorithmSha256 = 'SHA-256';
    case algorithmSha384 = 'SHA-384';
    case algorithmSha512 = 'SHA-512';
```

Our coding style uses lower camel case for enum cases.

Reply:
Or as by @Toflar s suggestion:

```suggestion
    case sha256 = 'SHA-256';
    case sha384 = 'SHA-384';
    case sha512 = 'SHA-512';
```